hs_rest_api/discovery.py

```diff
@@ -13,7 +13,7 @@ class DiscoveryResourceSerializer(HaystackSerializer):
     class Meta:
         index_classes = [BaseResourceIndex]
         fields = [
-            "text",
+            "title",
```

Comment:
should the validator be updated as well?

Reply:
Yes. I missed that. Text is a useless field to return; its only use is to be indexed.

Reply:
Actually, the validator searches text, and text is indexed but not stored. No validator update is needed.